deploy/k8s_deployments/ubiquity_logrotate

```diff
@@ -1,4 +1,4 @@
-/var/tmp/ubiquity-k8s-flex.log {
+/usr/libexec/kubernetes/kubelet-plugins/volume/exec/ibm~ubiquity-k8s-flex/ubiquity-k8s-flex.log {
```

Comment:
again since its inside the flex POD, the right directory is :
 /mnt/flex/ibm~ubiquity-k8s-flex/ubiquity-k8s-flex.log

So please use it